src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -188,7 +188,8 @@ export default function SidebarLinks() {
           state={{
             fromGameCard: false,
             runner: runner,
-            hasCloudSave: gameInfo?.cloud_save_enabled
+            hasCloudSave: gameInfo?.cloud_save_enabled,
+            gameInfo: gameInfo
```

Comment:
Since we now pass all `gameInfo`, I think the `hasCloudSave` property could be removed

Reply:
done